bosh/releases/pre_render_scripts/uaa/uaa/jobs/patch_pre-start.sh

```diff
@@ -36,6 +36,11 @@ patch --verbose "${target}" <<'EOT'
 +        mv /var/lib/ca-certificates/ca-bundle.pem /etc/ssl/certs/"$(basename "${OS_CERTS_FILE}")"
 +      ;;
 +
++      *rhel|centos*)
++        timeout --signal=KILL 180s /usr/bin/update-ca-trust -f -v
```

Comment:
```suggestion
+        timeout --signal=KILL 180s /usr/bin/update-ca-trust
```
`/usr/bin/update-ca-trust` doest not appear to take any arguments.